tests/testcases/sort/test_01/config.yaml

```diff
@@ -0,0 +1,4 @@
+function: "sort"
+expression: "ANN['Consequence']"
+chunk-size: "1"
+preserve-annotation-order: true
```

Comment:
I'd like the tests to have more descriptive names!

Reply:
done!